### ISSUE TITLE
[Keyboard Navigation– LCS – Index Analysis]: Extra tab encountered while navigating on controls under 'index analysis' pane

### ISSUE THREAD

**Issue by mohit406** [highcharts/highcharts#15540]
### User Experience: 

If Extra tab encountered while navigating on the page, then keyboard user will find hard to understand the actual flow of tab and the actual feature of the page which degrades the user.

Note:
User credentials should NOT be included in the bug.

### Repro Steps:

1. Open Url :https://lcs.dynamics.com
2. Login with valid credentials.
3. Select "Accessibility Review - D365FOE (Microsoft Corporation)" Project.
4. Project dashboard screen starts appearing.
5. Navigate to "Environment" section and select "Full details" button.
6. New screen starts appearing. Navigate to "Monitoring" section of Main page and select "Environment Monitoring".
7. New screen starts appearing where "Overview" tab is selected. Select "SQL Insights" tab.
8. New screen starts appearing where "Live View" tab is selected. Select "Index Analysis" tab.
9. Verify the issue.

### Actual Result:

Extra tab encountered while navigating on controls from 'index usage to graph' under 'index analysis' pane.

### Expected Result:

Extra tab should not encountered while navigating on controls from 'index usage to graph' under 'index analysis' pane. Focus order sequence should be 'index usage->graph->graph bar->user scans control.

Note:
This issue also exists with Firefox.

### MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7b0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7d

### 
### Reference Links

- Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
- Please reach out to C&AI Teams channel for any process related queries

![MAS2 4 3_Tab encountered](https://user-images.githubusercontent.com/82201935/115379084-d30c0200-a1ee-11eb-9239-7ff9da62780b.jpg)

**Comment by mohit406** [comment]
A11yMAS,A11ysev2,Accessibility,Benchmark,Benchmark-HCL-LCS-Feb2021,HCL,K4W,MAS2.4.3,Tabstops,Win10-Chrome

**Comment by oysteinmoseng** [comment]
Hi @mohit406, we do not have access to this page and are unable to debug the issue. Highcharts as of the latest version does not require additional keypresses for focus, and will set the focus to the first element in the chart immediately when tabbing to it. Are you not using the latest version?

**Comment by oysteinmoseng** [comment]
Closing this for now. Feel free to comment to reopen.
